src/GraphQL.Tests/Utilities/DateTimeOffsetTypeTests.cs

```diff
@@ -0,0 +1,56 @@
+// <copyright file="DateTimeOffsetTypeTests.cs" company="eVote">
+//   Copyright © eVote
+// </copyright>
```

Comment:
This creeped in again.

Reply:
This is a Resharper auto-generated file template. I really don/t want to have rights on that file :).
Removed.